src/app/containers/Timestamp/index.jsx

```diff
@@ -6,35 +6,37 @@ import relativeTime from './relativeTimestamp';
 // if the date is invalid return null - https://stackoverflow.com/questions/1353684/detecting-an-invalid-date-date-instance-in-javascript#answer-1353711
 const isValidDateTime = dateTime => !isNaN(dateTime); // eslint-disable-line no-restricted-globals
 
+const leadingZero = val => (val < 10 ? `0${val}` : val);
+
+const months = [ 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December' ]; // eslint-disable-line
+
+// 2019-03-22
 const longNumeric = {
-  locale: 'en-GB',
-  month: '2-digit',
-  day: '2-digit',
+  month: date => leadingZero(date.getMonth() + 1),
+  day: date => leadingZero(date.getDate()),
+  year: date => date.getFullYear(),
   reverse: true,
   separator: '-',
 };
 
+// 22 March 2019
 const shortAlphaNumeric = {
-  locale: 'en-GB',
-  month: 'long',
-  day: 'numeric',
+  month: date => months[date.getMonth()],
+  day: date => date.getDate(),
+  year: date => date.getFullYear(),
   reverse: false,
   separator: ' ',
 };
 
 const formatUnixTimestamp = (milliseconds, formatType) => {
   const dateObj = new Date(milliseconds);
-  const fullYear = dateObj.getFullYear();
-  const month = dateObj.toLocaleDateString('en-GB', {
-    month: formatType.month,
-  });
-  const day = dateObj.toLocaleDateString('en-GB', {
-    day: formatType.day,
-  });
+  const year = formatType.year(dateObj);
+  const month = formatType.month(dateObj);
+  const day = formatType.day(dateObj);
 
   const orderedDate = formatType.reverse
-    ? [day, month, fullYear].reverse()
-    : [day, month, fullYear];
+    ? [year, month, day]
```

Comment:
Instead of having a ternary here, I think it would be much more readable to define things explictly. e.g. in `shortAlphaNumeric` remove the `reverse: true` `reverse: false` and explicitly add a `format: [year, month, day]`.

Reply:
Chatted IRL and pushed change